src/pages/components/select.mdx

```diff
@@ -22,12 +22,12 @@ components:
   <Misuse>
     <ul>
       <li>
-        To filter a large list of options, consider an{' '}
-        <a href="/components/autocomplete">Autocomplete</a>
+        To filter a large list of options, use <a href="/components/autocomplete">Autocomplete</a>{' '}
+        instead
       </li>
       <li>
-        To make multiple selections from a list, use a{' '}
-        <a href="/components/multiselect">Multiselect</a>
+        To make multiple selections from a list, use{' '}
+        <a href="/components/multiselect">Multiselect</a> instead
```

Comment:
Just a reminder comment... Use `<Markdown>` if you prefer to use `[link](url)` style syntax for links.